src/System.IO/tests/MemoryStream/MemoryStreamTests.netcoreapp.cs

```diff
@@ -63,6 +63,7 @@ public void ReadSpan_DataReadAndPositionUpdated_Success()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Mono]
```

Comment:
Why is it disabled?

Reply:
Added comment (it requires Stream base class span API)